dev-python/ubelt/ubelt-0.8.8.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="A stdlib like feel, and extra batteries. Hashing, Caching, Timing, Progress"
+HOMEPAGE="https://github.com/Erotemic/ubelt"
+SRC_URI="https://github.com/Erotemic/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+IUSE="doc"
+
+SLOT="0"
+LICENSE="Apache-2.0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-python/six[${PYTHON_USEDEP}]
+	dev-python/ordered-set[${PYTHON_USEDEP}]"
+
+distutils_enable_tests pytest
```

Comment:
This on the other hand...
```
>>> Source compiled.
>>> Test phase: dev-python/ubelt-0.8.8
 * python3_6: running distutils-r1_run_phase python_test
ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --xdoctest --xdoctest-style=google
  inifile: /tmp/portage/dev-python/ubelt-0.8.8/work/ubelt-0.8.8/pytest.ini
  rootdir: /tmp/portage/dev-python/ubelt-0.8.8/work/ubelt-0.8.8

 * ERROR: dev-python/ubelt-0.8.8::testworld failed (test phase):
 *   Tests fail with python3.6
```

Reply:
Hmm, sorry about that, this ebuild is a mess. Don't know what happened here, maybe I just forgot about it.

Anyway I fixed it now, will push it after I fixed up the other ebuilds as well